microsoft-edge/progressive-web-apps-chromium/how-to/notifications-badges.md

```diff
@@ -192,7 +192,7 @@ self.addEventListener('notificationclick', event => {
 }, false);
 ```
 
-To learn more about notification actions, see [NotificationAction](https://developer.mozilla.org/docs/Web/API/NotificationAction) at MDN.
+To learn more about notification actions, see [NotificationAction](https://developer.mozilla.org/docs/Web/API/notification/actions) at MDN.
```

Comment:
```suggestion
To learn more about notification actions, see [Notification.actions](https://developer.mozilla.org/docs/Web/API/notification/actions) at MDN.
```

Reply:
@captainbrosset - fixes 404 at bottom of https://docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/notifications-badges#add-actions-to-notifications